### PR TITLE
Adding of more CRUD operations for app-gw subitems

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**
* address-pool
* frontend-ip
* frontend-port
* http-settings
* probe
* ssl-cert

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@EvgenyAgafonchikov__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: lib/commands/arm/network/appGateway._js [hunk 1]
```diff
@@ -333,6 +333,77 @@ __.extend(AppGateways.prototype, {
...
+      throw new Error(util.format($('Application gateway "%s" not found in resource group "%s"'), appGatewayName, resourceGroup));
+    }
+
+    var sslCert = utils.findFirstCaseIgnore(appGateway.sslCertificates, {name: sslCertName});
+    if (!sslCert) {
+      throw new Error(util.format($('A SSL certificate with name "%s" not found in the application gateway "%s"'), sslCertName, appGatewayName));
+    }
+
+    var index = utils.indexOfCaseIgnore(appGateway.sslCertificates, {name: sslCertName});
+
```
Comment (MikhailTryakhov) [inline lib/commands/arm/network/appGateway._js]: Please remove empty line

**Comment by MikhailTryakhov** [comment]
LGTM

**Comment by amarzavery** [comment]
@MikhailTryakhov - resolve conflicts

**Comment by amarzavery** [comment]
@MikhailTryakhov - resolve conflicts

**Comment by EvgenyAgafonchikov** [comment]
conflicts are resolved

**Pull request merged**